### PR TITLE
Fixes for e2e tests

### PR DESCRIPTION
**Pull request by mjura**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**
Issue #

**Special notes for your reviewer**:

**Checklist**:

- [ ] squashed commits into logical changes
- [ ] includes documentation
- [ ] adds unit tests
- [ ] adds or updates e2e tests
- [ ] backport needed

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,7 +8,7 @@ GIT_COMMIT?=$(shell git rev-parse HEAD)
 GIT_COMMIT_SHORT?=$(shell git rev-parse --short HEAD)
 GIT_TAG?=$(shell git describe --abbrev=0 --tags 2>/dev/null || echo "v0.0.0" )
 TAG?=${GIT_TAG}-${GIT_COMMIT_SHORT}
-REPO?=ghcr.io/rancher/eks-operator
+REPO?=docker.io/rancher/eks-operator
```
Comment (furkatgofurov7) [inline Makefile:11]: Can you elaborate more why this is needed?

File: test/e2e/suite_test.go [hunk 2]
```diff
@@ -138,6 +138,8 @@ var _ = BeforeSuite(func() {
 					"replicas=1",
 					"--set", fmt.Sprintf("hostname=%s.%s", e2eCfg.ExternalIP, e2eCfg.MagicDNS),
 					"--create-namespace",
+					"--devel",
+					"--set rancherImageTag=v2.9-head",
```
Comment (furkatgofurov7) [inline test/e2e/suite_test.go:142]: as is it throws an error:
```
[flag: --set rancherImageTag
      : exit status 1:Error: unknown flag: --set rancherImageTag
      
      {](msg: "[helm -n cattle-system install --set bootstrapPassword=admin --set global.cattle.psp.enabled=false --set extraEnv[0].name=CATTLE_SKIP_HOSTED_CLUSTER_CHART_INSTALLATION --set-string extraEnv[0].value=true --set replicas=1 --set hostname=172.18.0.3.sslip.io --create-namespace --devel --set rancherImageTag=v2.9-head rancher https://releases.rancher.com/server-charts/latest/rancher-2.8.2.tgz] cmd, failed with the following error: Error: unknown flag: --set rancherImageTag\n",)c
```

to fix it, please change it to:

```suggestion
					"--set",
					"rancherImageTag=v2.9-head",
```

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]

**Review by furkatgofurov7 — DISMISSED** [review]

**Comment by furkatgofurov7** [comment]
Pending open comment: https://github.com/rancher/eks-operator/pull/376#discussion_r1502333972

File: test/e2e/suite_test.go (hunk 2 above)
Comment (mjura) [reply test/e2e/suite_test.go]: I fixed it once again, now it is ok

**Review by alexander-demicev — APPROVED** [review]

**Review by furkatgofurov7 — APPROVED** [review]
Let's merge to get back CI to green, we can follow-up if needed.

**Comment by furkatgofurov7** [comment]
@mjura please open a backport PR to release-2.9 as well, thanks!

**Pull request merged**